pkg/clusterfile/clusterfile.go

```diff
@@ -74,7 +89,18 @@ func (c *ClusterFile) CommitSnapshot() {
 	*c.cfSnapshot = *c.cluster
```

Comment:
Although this is not part of this PR, i want to say the cfSnapshot maybe a bad design.

Why we need a snapshot of a cluster file? I think the only trusty cluster status is not the clusterfile but the k8s, so i think we no need control the version of clusterfile carefully.